src/patternfly/components/Radio/radio.scss

```diff
@@ -8,7 +8,7 @@
   --#{$radio}__label--FontSize: var(--#{$pf-global}--FontSize--md);
   --#{$radio}__label--LineHeight: var(--#{$pf-global}--LineHeight--sm);
   --#{$radio}__input--Height: var(--#{$radio}__label--FontSize);
-  --#{$radio}__input--MarginTop: calc(((var(--#{$radio}__label--FontSize) * var(--#{$radio}__label--LineHeight)) - var(--#{$radio}__input--Height)) / 2);
+  --#{$radio}__input--TranslateY: calc(((var(--#{$radio}__label--FontSize) * var(--#{$radio}__label--LineHeight)) - var(--#{$radio}__input--Height)) / 2);
```

Comment:
We really should be including comments in the code to document stuff like that so that in the future we can more easily know what's going on and why this was necessary.